nuget/spec/dependabot/nuget/file_parser_spec.rb

```diff
@@ -258,7 +258,7 @@
 
         it "has the right details" do
           expect(dependency).to be_a(Dependabot::Dependency)
-          expect(dependency.name).to eq("System.WebCralwer")
+          expect(dependency.name).to eq("System.WebCrawler")
```

Comment:
Hm this might be another one to keep, looks like this is coming from a downloaded manifest